carlos advice

CarlosSengo opened the pull request:
I would like some feedback

acodeinprogress commented:
Hey @CarlosSengo,

This looks great! I just have 1 small request before I can merge this in.

It looks like you added an extra line "104-advise" in the index.html file. If you could remove that, that would be great! Once you do that, you can add, commit, and push and your changes will join your pull request here. If no changes are made in 10 days, this pull request will close.

xoxo,
Sharon

CarlosSengo commented:
Hi Sharon, 

I updated index.html in my text editor, saved it and submitted it again. However during the process I was asked my ssh key in the terminal. I'm not sure what went wrong here. But I think the file is now updated. Can you please have a look? Also, was it supposed to ask me again for ssh key? If so, where can i find it? I'm still very fresh on git, need to review the lesson again :) thanks for your help

regards
carlos

acodeinprogress commented:
Hey @CarlosSengo,

Thanks for making that change! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon
